Makefile

```
@@ -148,6 +148,19 @@ binary: ## Build binary
 	@mkdir -p bin
 	CGO_ENABLED=0 $(GOBUILD) -ldflags="$(LDFLAGS)" -o bin/$(PROJECT_NAME) ./cmd/adapter
 
+.PHONY: install
+install: ## Build and install binary to GOPATH/bin
```

checkmake (0.2.2)

[warning] Target body for "install" exceeds allowed length of 5 (10).

(maxbodylength)